testdata/scripts/extract.txt

```diff
@@ -825,7 +882,6 @@ curl -X PUT \
 
 ### Responses
 
-<!-- TODO: add example -->
```

Comment:
this can be handled in a separate PR but we should probably figure out how better to communicate that a response is blank

Reply:
maybe we should print "response body is empty" or something, I think we should do it later.